build_tools/waf_dynamo.py

```diff
@@ -44,7 +44,7 @@ def transform_runnable_path(cls, platform, path):
 
 def platform_supports_feature(platform, feature, data):
     if feature == 'vulkan':
-        return platform not in ['win32', 'x86_64-win32', 'x86_64-linux', 'js-web', 'wasm-web']
+        return platform not in ['js-web', 'wasm-web']
```

Comment:
My mistake for sure. But not really sure why they ended up as "not supported". :/